pkg/cmd/repo/create/create.go

```diff
@@ -112,6 +112,9 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 
 			# create a remote repository from the current directory
 			gh repo create my-project --private --source=. --remote=upstream
+
+			# create a remote repository in an organization
+			gh repo create org-name/repo-name
```

Comment:
This seems duplicative of lines 110 & 111. Which is better, do you think, this change or the original example, and why?